File: 040-X/christer/christer.py
# import time 

# def fib(n): return 1 if n<2 else fib(n-1) + fib(n-2)

# start = time.time()
# print(fib(25))
# print(time.time()-start)

def ass(a,b): 
	if a!=b: print('Assert failed',a,b)

arr = [3,4]
hash = {'a':1, 'b':2}

ass(arr,[3,4])
ass(arr[1],4)
ass(hash,{'a':1, 'b':2})
ass(hash['a'],1)

arr1 = []
hash1 = {}

for value in arr:
	arr1.append(value)

for key in hash:
	hash1[key] = hash[key]

ass(arr,arr1)
ass(hash,hash1)

print('ok')

File: 040-X/christer/christer.pas
program christer;

uses SysUtils, epiktimer;

function fib(n:int32):int32;
begin
	if n<2 then fib := n else fib := fib(n-1) + fib(n-2)
end;

var
	ET: TEpikTimer;

procedure InitTimer;
begin
  ET := TEpikTimer.Create();
end;

var z: int32 = 0;
var i: int32;
var antal:int32 = 0;
begin
	InitTimer;
	ET.Clear; // optional... timer is cleared at creation
  ET.Start;
	//writeln(TimeToStr(Time));
	for i:=1 to 1
	 do begin
		z := fib(25);
		// antal := antal+1;
	end;
	writeln(z);
	//writeln(TimeToStr(Time));
	ET.stop;
	writeln(ET.Elapsed);
end.